qiskit_experiments/framework/experiment_data.py

```diff
@@ -24,21 +24,17 @@
 class ExperimentData(DbExperimentDataV1):
     """Qiskit Experiments Data container class"""
 
-    def __init__(
-        self,
-        experiment=None,
-        backend=None,
-        job_ids=None,
-    ):
+    def __init__(self, experiment=None, backend=None, job_ids=None, parent_id=None, root_id=None):
         """Initialize experiment data.
 
         Args:
             experiment (BaseExperiment): Optional, experiment object that generated the data.
             backend (Backend): Optional, Backend the experiment runs on.
             job_ids (list[str]): Optional, IDs of jobs submitted for the experiment.
-
-        Raises:
-            ExperimentError: If an input argument is invalid.
+            parent_id (str): Optional, ID of the parent experiment data
+                in the setting of a composite experiment
+            root_id (str): Optional, ID of the root experiment data
```

Comment:
This variable doesn't seem to be necessary, isn't it just the regular experiment id?